bin/docker-operations.py

```diff
@@ -22,32 +22,84 @@
 }
 
 
+def login_registry(docker_client: docker, registry: str, username: str, password: str):
+    # Login to Registry
+    print("INFO: Login to: " + registry)
+    docker_client.login(username=username, password=password, registry=registry)
+
+    return docker_client
+
+
 def get_image_tags(project_path: str):
     docker_image_path = root_directory / project_path
 
     try:
         with open(docker_image_path / "version.txt") as version_file:
-            version = version_file.read().strip()
-            if not semver(version).release:
-                raise Exception(
-                    f"ERROR: No valid semver found in {docker_image_path}/version.txt"
-                )
+            versions_text = version_file.read().strip()
+            versions = versions_text.split(",")
+
+            for version in versions:
+
+                if not semver(version).release:
+                    raise Exception(
+                        f"ERROR: No valid semver found in {docker_image_path}/version.txt"
+                    )
+
+            return versions
 
-            return version
     except FileNotFoundError:
         raise Exception(f"ERROR: version.txt not found in {docker_image_path}")
 
 
+def validate_tags(
+    docker_client: docker,
+    registry: str,
+    repository: str,
+    image: str,
+    tags: list,
+    override_tags: bool,
+):
+    docker_image_uri = registry + "/" + repository + "/" + image
+
+    if override_tags:
+        print(
+            "INFO: Override is set to True, if the tag will already present it will get override."
+        )
+        return tags
+    else:
+
+        final_tags = []
+        for tag in tags:
+
+            if tag == "latest":
+                print("INFO: The image tag is `latest`. It will override.")
+                final_tags.append(tag)
+            else:
+                try:
+                    docker_client.images.get_registry_data(
+                        name=(docker_image_uri + ":" + tag)
+                    )
+
+                    print(
+                        f"INFO: The docker tag {docker_image_uri}:{tag} already exists. Skipping the Docker push!"
+                    )
+                except APIError as dokerAPIError:
+                    print(
+                        f"INFO: Image tag {docker_image_uri}:{tag} not found on server. It will add to push list."
```

Comment:
Consistent wording with the prior INFO message
```suggestion
                        f"INFO: Docker tag {docker_image_uri}:{tag} not found on server. It will be added to the push list."
```